from finance_aggregator.finance_data.errors.configuration_error import (
    ConfigurationError,
)
